docs/home.md

```diff
@@ -57,93 +84,135 @@ underlying [Marinade Finance protocol](https://docs.marinade.finance/faq/faq#wha
-as well as the stability of the Solana blockchain.
+_NOTE_: Sunrise Stake has future plans to integrate other non-custodial staking protocols (in addition to Marinade and SolBlaze).
+ 
+##### Stake Pools 
+When Treasury Controller stakes your SOL in Marinade's stake pool, Marinade mints an amount of msOL tokens (Marinade Staked SOL tokens) representing the size of your holding of the stake pool, which is redeemable for SOL.
 
-### I am interested in integrating gSOL into my project, how can I do that?
+For example, if Marinade's stake pool contains 20 SOL and the total minted mSOL is 10, then if you add 1 SOl, Marinade mints something like (1/20) * 10 = 0.5 mSOL. 
+The stake pool size is now 21, you hold 0.5 out of a total 10.5 mSOL, so you own (0.5 / 10.5) of the pool of 21 sol, which equals 1 SOL.
 
-gSOL is a standard SPL token, and can be integrated into any project that supports SPL tokens.
+The staking process for SolBlaze is the same as above; when the Treasury Controller program stakes your SOL into BlazeStake's stake pool, Sunrise receives bSOL tokens (BlazeStake Staked SOL tokens) relative to your SOL's size of the stake pool. Just like mSOL, bSOL is redeemable for SOL.
 
-The gSOL token address is `gso12BFMxXD7RJaTUFfZZyuLW32M1hA9LoLZN7u9Und`.
+mSOL and bSOL accrue value 
 
-You can convert the gSOL back to SOL either by unstaking it manually
-in the [Sunrise Stake app](https://app.sunrisestake.com), through the upcoming JS library, or directly
-via the protocol itself.
+##### Liquidity Pool 
+2.  A  mSOL/SOL unstake pool - through which Sunrise becomes a liquidity provider to Marinade
+The liquidity pool is made up of SOL, mSOL and Liquidity Pool tokens
 
-### I am a validator, how can I get involved?
+  
+#### Step 3: Accruing yield
+###### Stake Pools
+At the end of each epoch (every 2 or 3 days), yield gets paid out into stake accounts, and the value of the overall stake pool increases. 
+As mSOL and bSOL represent a share of their respective stake pools, the value of mSol and bSOL also increases.
 
-In conjunction with the [Solana Foundation](https://solana.org/), we are exploring
-establishing a green stake pool, composed of validators that use renewable energy for
-their operations.
+The yield acrrued on the staked SOL, is equal to the value of mSOL and bSOL held by Sunrise minus the value of SOL staked (calculated as the circulatinbg supply of gSOL).
 
-Once the pool is established, Sunrise will direct stake to this pool, resulting in positive
-climate impact on both sides of the transaction:
+###### Liquidity Pool
 
-- Validators have an incentive to use renewable energy, reducing the carbon cost of the Solana network.
-- Stake yield is directed towards offsetting carbon emissions outside the solana network.
 
-### Where does the earned yield go?
+### Unstaking
+If you unstake your SOL, Sunrise calculates how much mSOL to sell back to Marinade to receive your SOL
```

Comment:
"If you unstake your SOL, Sunrise calculates how much of its share of the underlying stake and liquidity pools to sell, in order to receive your SOL"